addons/mrp/models/mrp_abstract_workorder.py

```diff
@@ -214,6 +221,18 @@ def _update_raw_moves(self):
         self.workorder_line_ids.unlink()
         self.env['stock.move.line'].create(vals_list)
 
+    def _strict_consumption_check(self):
+        mode = self._strict_consumption_mode()
```

Comment:
I don't get the purpose of this. Why the work order could produce less than planned?